src/scripts/dnf.sh

```diff
@@ -16,6 +16,9 @@ OSB_IFS=$IFS
 #   @2: Comma-separated list of comp-groups to install.
 #   @3: 0 or 1 to enable or disable --allowerasing when installing packages.
 #       Disabled by default. (optional)
+#   @4: 0 or 1 to enable or disable --nobest when installing packages.
+#       Disabled by default. (optional)
+#   @5: Comma-separated list of repository IDs to enable. (optional)
 #
```

Comment:
Feels like some assembly language, consider setting them to properly named variables.